charts/elasticsearch-umbrella/charts/elasticsearch-deployment/templates/hpa.yaml

```diff
@@ -0,0 +1,51 @@
+{{- if .Values.autoscaling.enabled }}
```

Comment:
Good to have in consideration the coordinating node in Elastic >6.8. For instance, for the Elasticsearch 7.17, this is some info about the [coordinating node](https://www.elastic.co/guide/en/elasticsearch/reference/7.17/modules-node.html#coordinating-node). 

Some conversations happened in the past regarding the need of coordinating nodes vs directly to data nodes.

Reply:
The "client" nodes would be a good candidate for this coordination nodes right @kaskol10 ?

Reply:
Based on this conversation, there're some settings that we would need to change (at some time):

https://www.elastic.co/guide/en/elasticsearch/reference/7.17/breaking-changes-7.9.html

The `node.{rol_name}` configurations are deprecated and they'll be removed in 8.X. Now it needs to be declared as an array (like the example about the coordination). Where now the rol should be defined as `node.roles:[]` and before it was just setting all the roles as false.